src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -80,6 +80,13 @@ export class ViewMappingProfile extends Component {
       // should resource be refreshed on POST/PUT/DELETE mutation
       shouldRefresh: () => false,
     },
+    marcFieldProtectionSettings: {
```

Comment:
This request is already defined in `<MappingProfiles>`, to avoid sending one more, consider getting its data from `parentResources` prop.